db/migrate/20170101185242_add_venue_id_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddVenueIdToEvents < ActiveRecord::Migration[5.0]
+  def change
+    add_column :events, :venue_id, :integer
```

Comment:
I would argue with that decision and I recommend you doing the same ;)
you don't need `nil` where it can be avoided and actually is bad to have, IMO.